javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,49 @@
+//A future age calculator
+const yearOfBirth = 1994;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("\"You will be "+ age +" years old in 2027\"" );
+
+//A dog age calculator
+
+const dogYearOfBirth = 2015;
+const dogYearFuture = 2025;
+const dogYearInHumanYear = dogYearFuture - dogYearOfBirth;
+const dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+const inHumanYear = '\"your dog will be '+dogYearInHumanYear+' human year old in 2025\" ';
+const inDogYear =  '\"your dog will be '+dogYear+' dog year old in 2025\" ';
+const shouldShowResultInDogYears = true;
+const actualold = shouldShowResultInDogYears === true ? inDogYear:inHumanYear;
+console.log(actualold);
+
+
+//A house price estimator
+
+const peterHouseVolume = 8*10*10;
+const juliaHouseVolume = 5*11*8;
+const peterGardenSizeInM2 = 100;
+const juliaGardenSizeInM2 = 70;
+const peterHouseCost = 2500000;
+const juliaHouseCosts = 1000000;
+const peterHousePrice = peterHouseVolume * 2.5 * 1000 + peterGardenSizeInM2 * 300;
+const juliaHousePrice = juliaHouseVolume * 2.5 * 1000 + juliaGardenSizeInM2 * 300;
```

Comment:
I think it is good idea if you can use variable instead of 2.5, 1000 and 300. 
It will also make sense why you use these values and for what?

Reply:
I have a doubt with this problem, [ housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300; ]  This is the equation that i have got  from homework question .How can i use variables instead of these values.
Thanks .

Reply:
just same values use variable like you already use.
for ex: let firstVal = 2.5;
let secondVal = 1000;
You can also use functions to calculate house cost.
const calculateHousePrice= (volume, val1, val2, gardenSize, val3) => {
const housePrice = peterHouseVolume * val1 * val2 + peterGardenSizeInM2 * val3;
retrun housePrice
}
then just it like
const peterHousePrice =  housePrice(peterHouseVolume, 2.5, 1000, peterGardenSizeInM2, 300);

Please use good naming for variables always.

Reply:
Thanks for the detailed explanation.
I have updated the file .
Thanks